packages/api/tests/test-es-indexer.js

```diff
@@ -0,0 +1,552 @@
+'use strict';
+
+const test = require('ava');
+const sinon = require('sinon');
+const fs = require('fs');
+const path = require('path');
+const aws = require('@cumulus/common/aws');
+const { StepFunction } = require('@cumulus/ingest/aws');
+const { randomString } = require('@cumulus/common/test-utils');
+const indexer = require('../es/indexer');
+const { Search } = require('../es/search');
+const { bootstrapElasticSearch } = require('../lambdas/bootstrap');
+const granuleSuccess = require('./data/granule_success.json');
+const granuleFailure = require('./data/granule_failed.json');
+const pdrFailure = require('./data/pdr_failure.json');
+const pdrSuccess = require('./data/pdr_success.json');
+
+const esIndex = randomString();
+process.env.bucket = randomString();
+process.env.stackName = 'my-stack';
+process.env.ES_INDEX = esIndex;
+let esClient;
+
+
+test.before(async () => {
+  // create the elasticsearch index and add mapping
+  await bootstrapElasticSearch('fakehost', esIndex);
+  esClient = await Search.es();
+
+  // create buckets
+  await aws.s3().createBucket({ Bucket: process.env.bucket }).promise();
+});
+
+test.after.always(async () => {
```

Comment:
Very small speed improvement, but these operations could be performed in parallel:

```
test.after.always(() =>
  Promise.all([
    esClient.indices.delete({ index: esIndex }),
    aws.recursivelyDeleteS3Bucket(process.env.bucket)
  ]));
```